### DISCUSSION TITLE
Features request

### DISCUSSION THREAD

**Discussion by krasnit** [slowscript/warpinator-windows#12] [Ideas]
I was looking for an alternative to Winpinator which is just too erratic in finding and keeping connections.  Your warpinator-windows works great!  
But I would like to see 2 additional features if possible.  The first is to set it to run in the background at Windows bootup.  That way it would be always running without having to get others to look for it and run it on their computers when file transfers are needed.  Yes, it can be added manually into the Windows startup folder, but only as long as one knows how to do that.  The second is the ability to automatically "reannounce" and "reconnect" itself regularly say every minute or so. This is particularly important for laptops which often go offline to conserve battery power.  It may seem like a simple thing to run this app and and click the reannounce and reconnect items, that is for computer savvies, but trying to explain that to others who barely understand the difference between a left and right mouse click can be quite challenging.
Keep up the good work though!

**Comment by slowscript** [comment]
Your request is noted. As for the second feature, what I will do is that it will detect the change in network connectivity and restart everything automatically on that event instead of with a fixed period. This has been on my To-Do list for a while. I am however currently focusing on the Android version (which doesn't have an alternative like this one) and some other projects so it might take some time before I get to it.
